internal/proxy/proxy.go

```diff
@@ -134,6 +141,8 @@ func (p *Proxy) Handle(w http.ResponseWriter, r *http.Request) {
 		requestType = "Jenkins UI"
```

Comment:
It's better to have metrics lable's value without space.  Also, I have seen to keep value in lower case always.

Reply:
Also, I understand that you want to count how many request comes from GitHub Web-Hook and from OSIO .. right .. ?
If so, why there is request type called 'Jenkins UI'.  Or it is NOT coming from OSIO?

Reply:
Well in code itself is identified as coming from Jenkins or coming from WebHook, so I decided to use same names.

Reply:
Well notice that these labels were already there, for jenkins-proxy a request is either a webhook or Jenkins UI so I decided to follow same convention so in metrics and in proxy itself maintains same name. But as you said I am going to put everything in lowercase and without space.